test/functional/apps/discover/_discover.js

```diff
@@ -19,20 +18,20 @@ bdd.describe('discover app', function describeIndexTests() {
     // delete .kibana index and update configDoc
     return esClient.deleteAndUpdateConfigDoc({'dateFormat:tz':'UTC', 'defaultIndex':'logstash-*'})
     .then(function loadkibanaIndexPattern() {
-      common.debug('load kibana index with default index pattern');
+      PageObjects.common.debug('load kibana index with default index pattern');
```

Comment:
Also, this doubles as a base class for the other Page Objects, so that they can use this generic functionality. The duplication of functionality seems like a code smell to me, but I'm having a hard time thinking of a more elegant solution. Each Page Object could import Common and keep a local instance of it, I guess? What do you think?